packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -233,14 +233,22 @@ class CodeSnippetDisplay extends MetadataDisplay<ICodeSnippetDisplayProps> {
     ];
   };
 
+  getDisplayName(metadata: IMetadata): string {
+    return `[${metadata.metadata.language}] ${metadata.display_name}`;
+  }
+
+  sortMetadata(): void {
+    this.props.metadata.sort((a, b) =>
+      this.getDisplayName(a).localeCompare(this.getDisplayName(b))
```

Comment:
We should ignore case like it is done in the JupyterLab File Browser

![image](https://user-images.githubusercontent.com/13068832/91220106-878d6c80-e6d0-11ea-9d60-32bfbc547b53.png)

Reply:
Is it not doing that, the lab sort code looks exactly the same as this.

Reply:
I just checked and it's already behaving that way
<img width="482" alt="Screen Shot 2020-08-25 at 12 58 41 PM" src="https://user-images.githubusercontent.com/13952758/91221620-c15f7280-e6d2-11ea-8c9e-87cb6880a9d7.png">

Reply:
Ah yes, my bad. I misread the MDN spec. It does appear to sort case insensitive by default.